packetbeat/docs/reference/configuration/packetbeat-options.asciidoc

```diff
@@ -659,6 +663,56 @@ at the end of the document.
 Note that limiting documents in this way means that they are no longer correctly
 formatted JSON objects.
 
+
+[[configuration-cassandra]]
+==== Cassandra Configuration Options
+
+The following settings are specific to the Cassandra protocol. Here is a sample
+configuration for the `cassandra` section of the +{beatname_lc}.yml+ config file:
+
+[source,yaml]
+------------------------------------------------------------------------------
+packetbeat.protocols.cassandra:
+  ports: [9042]
+  send_request: true
+  send_request_header: true
+  send_response: true
+  send_response_header: true
+  compressor: "snappy"
+  ignored_ops: ["SUPPORTED","OPTIONS"]
+------------------------------------------------------------------------------
+
+
+===== send_request
+
+If this option is enabled, the raw message of the request (`cassandra_request` field)
+is included in published events. The default is true.
+
+===== send_request_header
+
+If this option is enabled, the raw message of the response (`cassandra_request.request_headers` field)
+is sent to Elasticsearch. The default is true. enable `send_request` first before enable this option.
+
+===== send_response
+
+If this option is enabled, the raw message of the response (`cassandra_response` field)
+is included in published events. The default is true.
+
+===== send_response_header
+
+If this option is enabled, the raw message of the response (`cassandra_response.response_headers` field)
+is included in published events. The default is true. enable `send_response` first before enable this option.
+
+===== ignored_ops
+
+This option indicates which Operator/Operators will be ignored.
+
+===== compressor
+
+If this option is enabled, and also the flag indicates that the frame body was compressed, and
+the stream will decode by the compressor, currently only support `snappy` , The default compressor is nil.
```

Comment:
`
If this option is enabled, and also the flag indicates that the frame body was compressed, and
the stream will decode by the compressor, currently only support `snappy` , The default compressor is nil.
`

It's unclear if this is a boolean or some string.

e.g.

`Configures the default compression algorithm being used to uncompress compressed frames by name. Currently only `snappy` is can be configured. By default no compressor is configured`.